### PR TITLE
Update with Member Meeting changes #60

### PR DESCRIPTION
**Pull request by kitblake**
The FAQ has been radically renovated resulting from the momentous Members Meeting. It needs a good read and editorial round. Ref #60. 

Notes
- Membership questions have all been moved to the Members section.
- Roadmap: this item needs some more beef.
- The comm channels do not include the new Telegram groups. They're still 'in beta' while we get new admins up to speed.
- All the subheadings (FAQ questions) will show as changed. It turns out that in Markdown backslashes don't render, so they're being used to make the stupid two-spaces markup for breaks visible.  \

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -1,112 +1,125 @@
...
 
-**Where is the whitepaper?**  
+**Where is the whitepaper?**  \
 [On Github (PDF)](http://docs.google.com/gview?url=https://github.com/rchain/reference/raw/master/docs/RChainWhitepaper.pdf).
 See also the [RChain Architecture](http://rchain-architecture.readthedocs.io).
 
-**Where is the roadmap?**  
-A snapshot of the platform [milestone definitions](https://docs.google.com/spreadsheets/d/1_cPts5BdoZVmbikDGUhkgkdQvkeXbP6aKNI31O0BV3E/edit?usp=sharing) is available for commentary. The team has not yet committed to specific schedule dates. One estimate is the first release, Mercury, will be available after the goal of $10M committed investment is achieved.
+**Where is the roadmap?**  \
+A snapshot of the platform [milestone definitions](https://docs.google.com/spreadsheets/d/1_cPts5BdoZVmbikDGUhkgkdQvkeXbP6aKNI31O0BV3E/edit?usp=sharing) is available for commentary. The team has not yet committed to specific schedule dates.
```
Comment (pyrocto) [inline faq.md]: That's fairly out-of-date and doesn't really break things down in the way they're actually being worked on. There is a deadline of one year from close of the private sale for the delivery of Mercury. Medha can point you to stuff on JIRA that describes what the plan is for Mercury and where we are in implementing it.  There's a graphic at the top of https://rchain.atlassian.net/wiki/spaces/CORE/overview that gives a 50,000-foot view.

File: faq.md [hunk 2]
```diff
@@ -1,112 +1,125 @@
...
 
-**What is the governance model?**  
-The Co-op currently has a five-person [Board of Directors](https://www.rchain.coop/coop-information-1#board-of-directors). The officers are Greg Meredith, President, and Evan Jensen, Secretary. The Co-op is a member-driven organization, and the membership drive will begin once the regulation-related filings are completed for all U.S. states [and other jurisdictions?]. Additional board members and officers may be added. A membership meeting is planned for October 2017 in which elections will be held. Join the [RChain Slack](http://slack.rchain.coop) to learn specifics as plans take shape.
+- 3 years: Greg Meredith, Vlad Zamfir, Ian Bloom
+- 2 years: Kenny Rowe, Evan Jensen, Alexandr Bulkin
+- 1 year: Navneet Suman, Hendrik Jan Hilbolling, David Currin
 
-**What is the Membership Program?**  
-A program is being developed, but some aspects depend on having a working RChain blockchain. Permission from each of the 50 US states is needed in order to charge a membership fee and this process should be complete in the 4th quarter of 2017.
+The Co-op officers are: Greg Meredith, President; Evan Jensen, Secretary; Lise Rice, Treasurer.
```
Comment (pyrocto) [inline faq.md]: Lise or Lisa?

File: faq.md [hunk 3]
```diff
@@ -1,112 +1,125 @@
...
+**What skills do I need to participate?**  \
+At this point the entire development focus is on the core platform. If you’re a seasoned developer willing to learn a new programming language (Rholang), have experience with functional programming, appreciate formal specifications, or have expertise in comms then please get in touch. Down the road more and varied skills will be needed. Programmers versed in other languages, web developers, designers, Javascript experts, system administrators, beta testers, and quality assurance people will be in demand. If you'd like to participate please [join as a Member](https://member.rchain.coop) and put yourself in the Talent Pool.
```
Comment (pyrocto) [inline faq.md]: It's not so much Rholang that they need for building the core as Scala.  The VM, storage, and networking layers are all scala.  There will eventually be code for Casper, the Rev wallet, and a few other things in Rholang, but the vast majority is Scala.

**Comment by kitblake** [comment]
Thanks for the feedback, it's incorporated. Still need to liaise with Medha about roadmap resources..

File: faq.md (hunk 2 above)
Comment (Ojimadu) [reply faq.md]: I think LIsa

**Pull request merged**